nexus/social_media/tasks.py

```diff
@@ -0,0 +1,32 @@
+# Standard Library
+from datetime import datetime
+
+# Third Party Stuff
+from django.conf import settings
+
+# nexus Stuff
+from nexus.celery import app
+# Nexus Stuff
+from nexus.social_media.models import Post
+from nexus.social_media.services import post_to_facebook
+
+
+@app.task(name='publish_posts_to_social_media')
+def publish_posts_to_social_media():
+    if settings.LIMIT_POSTS is True and \
+       isinstance(settings.MAX_POSTS_AT_ONCE, int) and \
```

Comment:
Ideally, this should be checked in settings and not here.

You should raise ConfigurationError if this is not an `int`.